packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import Stack, { StackProps } from '@mui/material/Stack';
-import TextField from '@mui/material/TextField';
+import MaterialTextField from '@mui/material/TextField';
```

Comment:
It seems that we do use `Mui` prefix if we need for these components for the cases we currently have. 🤔 
```suggestion
import MuiTextField from '@mui/material/TextField';
```